nodepool/nl01.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -233,359 +233,7 @@ providers:
                   sudo: ALL=(ALL) NOPASSWD:ALL
 
-  - name: limestone-us-dfw-1
-    cloud: limestone
-    region-name: us-dfw-1
-    boot-timeout: 300
-    launch-timeout: 300
-    rate: 0.001
-    diskimages: &provider_limestone_diskimages
-      - name: centos-7
-        config-drive: true
-      - name: centos-8
-        config-drive: true
-      - name: fedora-32
-        config-drive: true
-      - name: ubuntu-bionic
-        config-drive: true
-      - name: ubuntu-xenial
-        config-drive: true
-    cloud-images: &provider_limestone_cloud-images
-      - name: asav9-12-3-20200302
-        config-drive: true
-        connection-type: network_cli
-      - name: ios-15.6-2T-20190524
-        config-drive: false
-        connection-type: network_cli
-      - name: iosxrv-6.1.3-20190604
-        config-drive: false
-      - name: iosxrv-7.0.2-20211703
-        config-drive: false
-        # NOTE(pabelanger): We don't have default gateway setup on boot, so
-        # this is an attempt to setup nodepool from doing ssh-keyscan.
-        connection-type: network_cli
-      - name: nxos-7.0.3.I5.1-20200505
-        config-drive: true
-        connection-port: 8022
-        connection-type: network_cli
-      - name: vEOS-4.20.10M-20190501
-        # NOTE(pabelanger): This seems to mess up the boot-loader for eos
-        config-drive: false
-        connection-type: network_cli
-      - name: vqfx-18.1R3-S2.5-20200116
-        config-drive: false
-      - name: vsrx3-18.4R1-S2.4-20190514
-        config-drive: false
-        connection-type: network_cli
-      - name: vyos-1.1.8-20190407
-        config-drive: true
-        connection-type: network_cli
-      - name: esxi-6.7.0-20190802001-STANDARD-20210409
-        config-drive: true
-        python-path: /usr/bin/python3
-      - name: VMware-VCSA-all-6.7.0-14836122-20201125
-        config-drive: true
-        python-path: /usr/bin/python3
-      - name: VMware-VCSA-all-7.0.1-16860138-20210217-3
-        config-drive: true
-        python-path: /usr/bin/python3
-      - name: VMware-VCSA-all-7.0.2-17694817-20210326
-        config-drive: true
-        python-path: /usr/bin/python3
-    pools:
-      # NOTE(pabelanger): Please contact pabelanger before making changes to
-      # this pool, especially increasing max-servers or changing labels. There
-      # is a real world financial cost in doing so that is charged to the
-      # Ansible org.
-      - name: s1.small
-        max-servers: 110
```

Comment:
Next time we have to do something like this we should probably set max-servers to 0 instead of removing the entire config.

Reply:
@goneri @dmsimard  Yes, please revert and set max-servers: 0 (don't actually believe we need to do that).

Also, not authing to limestone should be fine, we'll just log failures in the debug logs.  Has anyone talked to logan-?

What does doing this fix?

Reply:
The two Limestone regions are broken since this morning. We cannot auth ourselves: `The request you have made requires authentication. (HTTP 401) (Request-ID: req-9a669fab-7622-4c0c-bb5f-b3e6a3a34fd2)
`. No logan does not answer.